src/wifi/model/dmg-wifi-mac.h

```diff
@@ -870,13 +870,14 @@ class DmgWifiMac : public RegularWifiMac
    * \param srcAddress The MAC address of the source station.
    * \param dstAddress The MAC address of the destination station.
    */
-  typedef void (* ServicePeriodCallback)(Mac48Address srcAddress, Mac48Address dstAddress);
-  TracedCallback<Mac48Address, Mac48Address> m_servicePeriodStartedCallback;
-  TracedCallback<Mac48Address, Mac48Address> m_servicePeriodEndedCallback;
+  typedef void (* ServicePeriodCallback)(Mac48Address srcAddress, Mac48Address dstAddress, bool isSource);
```

Comment:
It can be helpful for easy debugging. What do you think? :)

Reply:
i'm not sure, how would you intend to use it? maybe i'm missing something

Reply:
It's just that the `ServicePeriodCallback` can be fired also when an SP starts at the destination node (of the SP). It might be useful when debugging with many STAs. It's not essential. If we don't want to modify too much existing modules let's avoid this.

Reply:
no I think it's fine, it's not the Wifi core module, if you think it's useful then let's keep it